dotcom-rendering/src/web/components/Card/Card.stories.tsx

```diff
@@ -9,9 +9,9 @@ import {
 import { breakpoints, from } from '@guardian/source-foundations';
 import { storiesOf } from '@storybook/react';
 import React from 'react';
-import { Card } from './Card';
+import { Card, Props as CardProps } from './Card';
```

Comment:
I always prefer to import types explicitly, but we don't have a linting rule, yet